platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch

```diff
@@ -185,15 +185,15 @@ index e4b1b0f1c..36cd6aacf 100644
              return "";
          }
 -        return String.format("%.2f",
-+        // -- GODOT start --
++        // -- BLAZIUM start --
 +        return String.format(Locale.ENGLISH, "%.2f",
                  (float) overallProgress / (1024.0f * 1024.0f))
                  + "MB /" +
 -                String.format("%.2f", (float) overallTotal /
 +                String.format(Locale.ENGLISH, "%.2f", (float) overallTotal /
```

Comment:
This line maybe from a different branch, please validate this is required for rebranding